wait_until_replica_is_caught_up_to_master.go

```diff
@@ -11,7 +10,7 @@ import (
 )
 
 type ReplicatedMasterPositionFetcher interface {
-	Current(*sql.DB) (mysql.Position, error)
+	Current(db *DB) (mysql.Position, error)
```

Comment:
Same comment here. I don't believe we need the named parameter to this interface method